mmcv/runner/hooks/lr_updater.py

```diff
@@ -201,8 +201,9 @@ def get_lr(self, runner, base_lr):
 @HOOKS.register_module
 class CosineLrUpdaterHook(LrUpdaterHook):
 
-    def __init__(self, target_lr=0, **kwargs):
-        self.target_lr = target_lr
+    def __init__(self, target=0, as_ratio=False, **kwargs):
```

Comment:
- We may rename the hook to `CosineAnnealingLrUpdaterHook`?
- Rename `target_lr` to `min_lr` to keep it consistent with PyTorch APIs.
- `min_lr=None, min_lr_ratio=None` instead of `target=0, as_ratio=False`
- We may add an argument `last_epoch=-1`.